core/db/migrate/20250129061658_add_metadata_to_spree_resources.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class AddMetadataToSpreeResources < ActiveRecord::Migration[7.2]
+  def change
+    # List of Resources to add metadata columns to
+    %i[
```

Comment:
Please apply metadata only to
      spree_users
      spree_orders
      spree_line_items
      spree_shipments
      spree_payments
      spree_refunds
      spree_customer_returns
      spree_store_credit_events
      spree_return_authorizations